docs/framework/common-client-technologies/index.md

```diff
@@ -1,16 +1,15 @@
 ---
-title: "Common Client Technologies in the .NET Framework"
+title: "Common client technologies - .NET Framework"
 ms.date: "03/30/2017"
 ms.assetid: 733fc580-337e-4b7c-9756-b70798c18bc3
 author: "mairaw"
 ms.author: "mairaw"
 ---
-# Common Client Technologies in the .NET Framework
+# Common client technologies in the .NET Framework
+
 This section describes different technologies that you can use in your client applications.  
```

Comment:
nit: it now only describes a single technology. I wonder if this portal page to a single technology is even useful.

Reply:
I was even thinking if the other technology is even still useful. Maybe I should retire that whole section?

Reply:
I would tend to agree -- the entire section could be archived.